project/Versions.scala

```diff
@@ -0,0 +1,5 @@
+object Versions {
```

Comment:
I totally agree, I do this in almost every build. I'm not sure if we would use it in this build however since the scala version is very important for us.